google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/it/BuiltinMetricsIT.java

```diff
@@ -58,64 +88,174 @@ public class BuiltinMetricsIT {
+            .setIsolationPolicy(AppProfile.StandardIsolationPolicy.of(AppProfile.Priority.LOW)));
+    instanceAdminClient.createAppProfile(
+        CreateAppProfileRequest.of(testEnvRule.env().getInstanceId(), appProfileDefault)
+            .setRoutingPolicy(
+                AppProfile.SingleClusterRoutingPolicy.of(testEnvRule.env().getPrimaryClusterId()))
+            .setIsolationPolicy(AppProfile.StandardIsolationPolicy.of(AppProfile.Priority.LOW)));
+
+    metricReader = InMemoryMetricReader.create();
+
+    SdkMeterProviderBuilder meterProvider =
+        SdkMeterProvider.builder().registerMetricReader(metricReader);
+    BuiltinMetricsView.registerBuiltinMetrics(testEnvRule.env().getProjectId(), meterProvider);
+    OpenTelemetry openTelemetry =
+        OpenTelemetrySdk.builder().setMeterProvider(meterProvider.build()).build();
+
+    BigtableDataSettings.Builder settings = testEnvRule.env().getDataClientSettings().toBuilder();
+
+    clientCustomOtel =
+        BigtableDataClient.create(
+            settings
+                .setMetricsProvider(CustomOpenTelemetryMetricsProvider.create(openTelemetry))
+                .setAppProfileId(appProfileCustomOtel)
+                .build());
+    clientDefault = BigtableDataClient.create(settings.setAppProfileId(appProfileDefault).build());
   }
 
-  @AfterClass
-  public static void tearDown() {
+  @After
+  public void tearDown() {
     if (metricClient != null) {
       metricClient.close();
     }
-    if (table != null) {
-      tableAdminClient.deleteTable(table.getId());
+    if (tableCustomOtel != null) {
+      tableAdminClient.deleteTable(tableCustomOtel.getId());
+    }
+    if (tableDefault != null) {
+      tableAdminClient.deleteTable(tableDefault.getId());
+    }
+    if (instanceAdminClient != null) {
+      instanceAdminClient.deleteAppProfile(
+          testEnvRule.env().getInstanceId(), appProfileCustomOtel, true);
+      instanceAdminClient.deleteAppProfile(
+          testEnvRule.env().getInstanceId(), appProfileDefault, true);
+    }
+    if (clientCustomOtel != null) {
+      clientCustomOtel.close();
+    }
+    if (clientDefault != null) {
+      clientDefault.close();
+    }
+  }
+
+  @Test
+  public void testBuiltinMetricsWithDefaultOTEL() throws Exception {
+    logger.info("Started testing builtin metrics with default OTEL");
+    tableDefault =
+        tableAdminClient.createTable(
+            CreateTableRequest.of(PrefixGenerator.newPrefix("BuiltinMetricsIT#test1"))
+                .addFamily("cf"));
+    logger.info("Create default table: " + tableDefault.getId());
+    clientDefault.mutateRow(
+        RowMutation.create(tableDefault.getId(), "a-new-key").setCell("cf", "q", "abc"));
+    ArrayList<Row> rows =
+        Lists.newArrayList(clientDefault.readRows(Query.create(tableDefault.getId()).limit(10)));
+
+    Stopwatch stopwatch = Stopwatch.createStarted();
```

Comment:
This needs a better name, reading over the code, I assumed that this measures actual rpc duration. 
Maybe something like metricPollingStopwatch with a comment of soemthing like "this stopwatch is used for to limit fetching of metric data in verifyMetrics